lib/utils/deprecation/config_file.rb

```diff
@@ -0,0 +1,108 @@
+require 'stringio'
+require 'json'
+require 'inspec/globals'
+
+module Inspec
+  module Deprecation
+    class ConfigFile
+      GroupEntry = Struct.new(:name, :action, :prefix, :suffix, :exit_status, :occurences)
+
+      # What actions may you specify to be taken when a deprecation is encountered?
+      VALID_ACTIONS = [
+        # Hard exit inspec, no stacktrace, exit code specified or DEFAULT_HARD_EXIT_STATUS
```

Comment:
I'm not sure DEFAULT_EXIT_STATUS is used - I think it just calls Inspec::UI.exit